### DISCUSSION TITLE
Changing 2 cars on same account to different accounts

### DISCUSSION THREAD

**Discussion by joao-afonso** [teslamate-org/teslamate#4094] [Q&A]
We have two cars in the family (let's call them cars 1 and 2), both in the same account (account A) and my wife and I just share the account. Teslamate has been working flawlessly for both cars since we first started using it, about 6 months ago. 

We now wanted to have separate Tesla accounts, each being the owner of one car, but being added as the driver of the other car. So account A will be the owner of car 2 and also be added as a driver of car 1 and account B will be the owner of car 1 and be added as a driver of car 2.

What would I need to do on Teslamate's side to keep everything working the same way, still keeping track of both cars while also keeping the history it already has?

**Comment by cwanja** [comment]
I _believe_ nothing, since you are adding both cars as secondary drivers. Assuming you want to keep a single TeslaMate instance going. 

Full disclosure - do not own two cars nor have two Tesla accounts. Going off memory from others that we have seen posted.

**Reply by joao-afonso** [reply]
Thank you for your answer. 

I've been looking at the documentation and this got me thinking: Are the car IDs going to change? Won't that cause issues? The original account, account A, will be the owner of the second car that was added, so I am assuming that is car ID number 2. After car number 1 is changed to the other account, isn't car number 2 going to become car ID number 1? And when I add account A as driver of what used to be car 1, won't it become car ID number 2?

**Reply by cwanja** [reply]
The internal ID is unique to TeslaMate. I believe we check against VIN and Tesla's ID which will remain the same regardless of what account the car is under.
